### ISSUE TITLE
Allow to override the default onClick event

### ISSUE THREAD

**Issue by danielo515** [wmira/react-sidenav#22]
Hello,

Thanks for this great NavBar. I am just exploring the surface and I'm not using all the features, but looks very promising. Currently I am using it for anchor navigation (within the same page)
This not a big problem because I found a workaround ( one of many possible) however, it would be cool if we discuss about it.

I am on the situation where, on a single menu all the elements shares the same selection callback logic except one. Having to write callback branching just for one element is a bit inconvenient. It would be awesome if particular elements could specify an specific selection callback that should override the one the whole navbar has.

My current workaround is to build TWO navbars, one after the other, that just look like one. I have disabled the selection highlight so the fact that are actually two navbars is not noticeable.  Other workaround may be to check the id on the callback and act differently depending on it.

Many thanks, and regards

**Comment by wmira** [comment]
@danielo515 thanks for the report. if i get what you are saying..
```javascript
<SideNav onItemSelection={}>
  <Nav id='1' onItemSelection={}>
 <Nav id='2'/>
</SideNav> 
```

You wanted when 1 is clicked to not trigger the main onItemSelection but only onItemSelection on Nav1?

**Comment by danielo515** [comment]
Yes, exactly . And when clicking Nav 2 (or 3 or 4) to trigger the main onItemSelection. 
Thanks and regards

**Comment by intersplice** [comment]
HI was this every implemented?
